ferrocene/tools/traceability-matrix/src/matrix.rs

```diff
@@ -286,6 +265,39 @@ impl Element {
             (None, None) => "no name".into(),
         }
     }
+
+    fn cli_option(matrix_page: &Page, option: &CliOption, link: String) -> Self {
```

Comment:
Because I think the original function (`fn prepare`) is just way to big to be understood quickly. Breaking this into multiple logical units makes it way more comprehensible, at least for me.